src/main/kotlin/org/rust/lang/utils/RustEscapesUtils.kt

```diff
@@ -32,7 +32,7 @@ fun parseRustStringCharacters(chars: String, outChars: StringBuilder, sourceOffs
 
     val outOffset = outChars.length
     var index = 0
-    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true)).forEach {
+    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true, true)).forEach {
```

Comment:
Why we use `extended` here and not [here](https://github.com/intellij-rust/intellij-rust/pull/296/files#diff-d2d81e2509892bcb970852e6f23f80d0R124) ?